sunbeam-python/sunbeam/jobs/common.py

```diff
@@ -279,6 +281,15 @@ def get_host_total_ram() -> int:
     raise Exception("Could not determine total RAM")
 
 
+def get_host_total_cores() -> int:
+    """Reads cpuinfo to get total cores."""
+    with open("/proc/cpuinfo") as f:
+        for line in f:
+            if line.startswith("cpu cores"):
+                return int(line.split()[-1])
+    raise Exception("Could not determine number of cores")
```

Comment:
Using `os.cpu_count()` is more straightforward (there's no equivalent for memory, hence why it's reading from meminfo)

```suggestion
def get_host_total_cores() -> int:
    """Reads cpuinfo to get total cores."""
    return os.cpu_count()
```

Reply:
Done